src/pages/jobs.md

```diff
@@ -15,26 +15,94 @@ We're hiring
 
 # About Shorebird
 
-## Why should I work at Shorebird?
+Shorebird is a small, venture-funded startup (est Jan 2023) trying to make
+Flutter the default way to write apps. So far we've built and shipped Code Push
+for Flutter to let businesses over the air update their apps instantly on users
+phones.
+
+## Why work at a startup?
+
+Having worked at both, I can tell you, startups feel very different from big
+companies and big teams.
+
+Working at a startup means:
+
+- Shipping. We ship 5 times a day, not 5 times a year.
```

Comment:
5 times a day is maybe a bit of a stretch :)

Reply:
True, I can change it to 5 times a week. Although sometimes we ship 3 times a day 🤣